### PR TITLE
test(Sidebar) : Should not crash the app, when the rhs is opened and user goes to and back from system console

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Is the feature in stable branch or under review?
- [ ] Feature in stable branch (master, main)?
- [X] Feature still under review? If yes, list all related pull requests for reference.
  https://github.com/mattermost/mattermost-webapp/pull/11635

#### Test client

- [X] Browser
- [ ] Mobile App (version, OS)
- [X] Desktop App (version, OS)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/23"></a>

### CODE REVIEW

File: data/test-cases/channels/sidebar/MM-X1.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+# (Required) Ensure all values are filled up
+name: "Should be able to open recent mentions, visit the system console and come back without issues"
+status: Active
+priority: Normal
+folder: Channels
+authors: ""
```
Comment (yasserfaraazkhan) [inline data/test-cases/channels/sidebar/MM-X1.md:7]: authors: "[@M-ZubairAhmed](https://github.com/M-ZubairAhmed)"

**Review by yasserfaraazkhan — APPROVED** [review]
Thanks @M-ZubairAhmed ,
you can add your GitHub username in `authors:`

File: data/test-cases/channels/sidebar/MM-X1.md (hunk 1 above)
Comment (M-ZubairAhmed) [reply data/test-cases/channels/sidebar/MM-X1.md]: added

File: data/test-cases/channels/sidebar/MM-X1.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# (Optional)
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+location: Sidebar
+component: null
+tags: []
+labels:
+- se-prod
+- channel
+- rhs
```
Comment (M-ZubairAhmed) [inline data/test-cases/channels/sidebar/MM-X1.md:19]: are these correct?

File: data/test-cases/channels/sidebar/MM-X1.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (M-ZubairAhmed) [inline data/test-cases/channels/sidebar/MM-X1.md:1]: is the test folder placement correct? Or do we want these to be somewhere else

Comment (saturninoabril) [reply data/test-cases/channels/sidebar/MM-X1.md:1]: I'm fine with this.

File: data/test-cases/channels/sidebar/MM-X1.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+# (Required) Ensure all values are filled up
+name: "Should be able to open recent mentions, visit the system console and come back without issues"
+status: Active
+priority: Normal
+folder: Channels
```
Comment (saturninoabril) [inline data/test-cases/channels/sidebar/MM-X1.md:6]:
```suggestion
folder: Sidebar
```

Also for other test cases.

File: data/test-cases/channels/sidebar/MM-X1.md (hunk 2 above)
Comment (saturninoabril) [inline data/test-cases/channels/sidebar/MM-X1.md:19]: "se-prod" meant there's Selenium test in production which we no longer do. Same for other test cases.
```suggestion
labels:
- channel
- rhs
```

File: data/test-cases/channels/sidebar/MM-X2.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+# (Required) Ensure all values are filled up
+name: "Should be able to open saved posts, visit the system console and come back without issues"
+status: Active
+priority: Normal
+folder: Channels
+authors: ""
```
Comment (saturninoabril) [inline data/test-cases/channels/sidebar/MM-X2.md:7]: Please add you GH handle. Same for other test cases.
```suggestion
authors: "@M-ZubairAhmed"
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Zubair! LGTM, except for few requests.

File: data/test-cases/channels/sidebar/MM-X1.md (hunk 4 above)
Comment (M-ZubairAhmed) [reply data/test-cases/channels/sidebar/MM-X1.md:6]: ✅  Updated as per request/discussion

File: data/test-cases/channels/sidebar/MM-X1.md (hunk 2 above)
Comment (M-ZubairAhmed) [reply data/test-cases/channels/sidebar/MM-X1.md:19]: ✅  Updated as per request/discussion

File: data/test-cases/channels/sidebar/MM-X2.md (hunk 5 above)
Comment (M-ZubairAhmed) [reply data/test-cases/channels/sidebar/MM-X2.md]: ✅  Updated as per request/discussion

**Review by saturninoabril — APPROVED** [review]
Thanks Zubair, LGTM.

**Comment by M-ZubairAhmed** [comment]
@saturninoabril @yasserfaraazkhan can you let me know the test ids for these.

**Comment by saturninoabril** [comment]
It's here - https://github.com/mattermost/mattermost-test-management/commit/8ca3dc3f680821bfa05400f9176f967f7b990f76

**Pull request merged**